### DISCUSSION TITLE
NPM packages for testing foundry

### DISCUSSION THREAD

**Discussion by kingmanas** [Cyfrin/foundry-full-course-cu#1163] [General]
Which npm packages are to be included for testing contracts

**Comment by vigneshbalu96** [comment]
Where are you actually facing the issue? npm packages are essential for the whole project in order to start and work with not only specific to testing.

**Comment by xyz899** [comment]
Hi,
Foundry is typically installed via a shell script using :
`curl -L https://foundry.paradigm.xyz | bash`
 (as seen in this command ) or by downloading pre-built binaries. It doesn't rely on Node.js or npm
